src/main/java/com/indemnity83/irontank/IronTank.java

```diff
@@ -27,8 +24,6 @@
 	@SidedProxy(clientSide = Reference.CLIENT_PROXY_CLASS, serverSide = Reference.SERVER_PROXY_CLASS)
 	public static IProxy proxy;
 
-	public static ItemMapHelper itemMapHelper;
```

Comment:
Why was this removed?

Reply:
This removed because I have reberted all the items id changes

Reply:
So now its the same id as before